### DISCUSSION TITLE
Signed Policy doesnt work in Docker in EC2 ok locally

### DISCUSSION THREAD

**Discussion by danrossi** [OvenMediaLabs/OvenMediaEngine#1649] [Help]
I am using this same config for the local and remote server. and I can't get publishing working. Any ideas ?

```
<SignedPolicy>
				<PolicyQueryKeyName>policy</PolicyQueryKeyName>
				<SignatureQueryKeyName>signature</SignatureQueryKeyName>
				<SecretKey>65r)9Fw0p2_</SecretKey>

				<Enables>
					<Providers>rtmp,webrtc,srt</Providers>
					<Publishers>webrtc,hls,llhls,dash,lldash</Publishers>
				</Enables>
			</SignedPolicy>
```

I run the command like

```
 ./simple_signed_policy_url_generator.sh "aKq#1kj" rtmp://ome.electroteque.org:1935/app/C6Lx6ku6FEXgKtt signature policy 31556952
 ```

**Comment by bchah** [comment]
In your example, the key you used in the command does not match the one in your configuration. What is the output from running the exact same command locally versus on EC2 - are they different? That may provide a clue.

**Reply by danrossi** [reply]
It's not a key thing. I am using the same key on the server. It's just an example command.

**Reply by getroot** [reply]
what was your final rtmp url? and upload your ovenmediaengine.log file
